assets/js/modules/reader-revenue-manager/components/dashboard/RRMSetupSuccessSubtleNotification.js

```diff
@@ -221,17 +269,130 @@ export default function RRMSetupSuccessSubtleNotification( {
 							{ ...gaTrackingProps }
 						/>
 					}
+					type="warning"
 				/>
 			</Notification>
 		);
 	}
 
-	if ( publicationOnboardingState === ONBOARDING_ACTION_REQUIRED ) {
+	if ( publicationOnboardingState === ONBOARDING_COMPLETE ) {
+		if ( isRRMV2Enabled ) {
+			const notificationContent = {
+				title: __(
+					'Success! Your Reader Revenue Manager account is set up',
+					'google-site-kit'
+				),
+				description: '',
+				primaryButton: {
+					text: __( 'Manage CTAs', 'google-site-kit' ),
+					ctaLink: `${ settingsURL }#connected-services/reader-revenue-manager/edit`,
+					isCTALinkExternal: false,
+				},
+				secondaryButton: {
+					text: __( 'Got it', 'google-site-kit' ),
+					onClick: dismissNotice,
+				},
+			};
+
+			switch ( paymentOption ) {
+				case 'subscriptions':
+					if ( productID === 'openaccess' ) {
+						notificationContent.description = __(
+							'You can edit your settings to manage product IDs and select which of your site’s pages will include a subscription CTA.',
+							'google-site-kit'
+						);
+					} else {
+						notificationContent.description = __(
+							'You can edit your settings and select which of your site’s pages will include subscription CTAs.',
+							'google-site-kit'
+						);
+					}
+					break;
+				case 'contributions':
+					if ( productIDs.length > 0 && productID === 'openaccess' ) {
+						notificationContent.description = __(
+							'You can edit your settings to manage product IDs and select which of your site’s pages will include a contribution CTA.',
+							'google-site-kit'
+						);
+					} else {
+						notificationContent.description = __(
+							'You can edit your settings and select which of your site’s pages will include contribution CTAs.',
```

Comment:
I think the copy here should refer to "a contribution CTA" in line with the other descriptions, with the AC updated accordingly (this variant doesn't appear to be in Figma to refer to or update).

```suggestion
							'You can edit your settings and select which of your site’s pages will include contribution CTAs.',
```

Reply:
Updated ACs and the PR accordingly, thanks Tom!